datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/SchemaTable.tsx

```diff
@@ -194,11 +193,8 @@ export default function SchemaTable({
                     columns={allColumns}
                     dataSource={rows}
                     rowKey="fieldPath"
-                    components={{
-                        body: {
-                            row: SchemaRow,
-                        },
-                    }}
+                    scroll={{ y: '100vh', x: '100%' }}
```

Comment:
unfortunately adding this `scroll` causes a double scroll in the table, which is not desired. this is what I'm seeing (a scrollbar inside of a scrollbar)
<img width="1465" alt="image" src="https://user-images.githubusercontent.com/28656603/198143571-f281ef50-3c8d-4a84-804a-2a2539aab51f.png">

In order to use the `scroll` prop here we'll have to rejigger some css elsewhere to make things look nice

Reply:
i'm also seeing some buggy behavior when you try to grab the scrollbar and drag it and then try to scroll afterward (I'm assuming a bug with ant-design's `scroll` prop here?)
here's a screen recording of what I'm talking about:

https://user-images.githubusercontent.com/28656603/198144639-0cb6e6ed-af0f-4242-b5ae-3de5e5492f8d.mov

Reply:
finally, on smaller tables, I'm seeing a scrollbar added unnecessarily and cutting off the final row from reaching the bottom:
<img width="1356" alt="image" src="https://user-images.githubusercontent.com/28656603/198145215-36b8bcf1-dc63-46b2-ab2e-8d1649f97d10.png">

Reply:
Ah! Good catch(es). Should be pretty straight forward to fix these 🙂

Reply:
awesome! thanks so much!